### PR TITLE
XW-2188 | SCSS global variables moved to separate files

### PR DESCRIPTION
**Pull request by weronika**

### CODE REVIEW

**Comment by kvas-damian** [comment]
Please resolve merge conflicts

**Comment by kvas-damian** [comment]
`style-guide/styles/wds-base/_icons.scss → dist/scss/wds-components/_icons.scss`
I'm not fully convinced that `icon` is a ` component`. If we believe so we should document it on RP.

File: style-guide/styles/wds-variables/_global-navigation.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+$border-top-width: 6px;
+$logo-width: 230px;
+$nav-background-color: $color-white;
+$nav-height: 49px;
+
+
```
Comment (kvas-damian) [inline style-guide/styles/wds-variables/_global-navigation.scss]: Newlines?

File: style-guide/styles/wds-variables/_buttons.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+$button-active-color: darken($color-link, 20%);
+$button-line-height: 1.3em;
+$hover-transition-duration: 300ms;
```
Comment (kvas-damian) [inline style-guide/styles/wds-variables/_buttons.scss]: Do we want to export it as DS variable? If so then probably we need to make the name more descriptive.

Comment (kvas-damian) [inline style-guide/styles/wds-variables/_buttons.scss]: Please rethink which variables should be exported from DS and mark rest as private variables of DS.

**Comment by kvas-damian** [comment]
Please as some description to PR and ping interested parties.

File: style-guide/styles/wds-variables/_typography.scss [hunk 3]
```diff
@@ -0,0 +1,18 @@
+// Font family - see https://www.smashingmagazine.com/2015/11/using-system-ui-fonts-practical-guide/
+$wds-font-family: 'San Francisco', 'Helvetica Neue', 'Roboto', 'Segoe UI', 'Ubuntu', 'Droid Sans', sans-serif;
+
+// Line height
+$wds-line-height: 1.5em;
+
+// Typescale sizes
+$wds-typescale-size-minus-3: 10px;
```
Comment (kvas-damian) [inline style-guide/styles/wds-variables/_typography.scss]: What do you think about updating documentation for those variables?
http://wikiadesignsystem.com/#/base-styles/typography

Comment (kvas-damian) [inline style-guide/styles/wds-variables/_typography.scss]: The same for colors: http://wikiadesignsystem.com/#/base-styles/colors
Please make sure documentation is up to date now.

File: dist/scss/wds-variables/_global-navigation.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+$wds-global-navigation-background-color: $wds-color-white;
+$wds-global-navigation-height: 55px;
+$wds-global-navigation-logo-width: 230px;
```
Comment (kvas-damian) [inline dist/scss/wds-variables/_global-navigation.scss]: Why do we need this variable public?
I can't find it used in other places than global navigation styles in DS.

File: dist/scss/wds-variables/_lists.scss [hunk 5]
```diff
@@ -0,0 +1 @@
+$wds-list-item-padding: 11px 0;
```
Comment (kvas-damian) [inline dist/scss/wds-variables/_lists.scss]: Did we considered making it a local variable used only in `_lists.scss`?

File: dist/scss/wds-variables/_dropdowns.scss [hunk 6]
```diff
@@ -0,0 +1 @@
+$wds-dropdown-content-z-index: 1;
```
Comment (kvas-damian) [inline dist/scss/wds-variables/_dropdowns.scss]: Please once again rethink scope of those variables. Do we really want encourage people to use this variables?
Can we find a valid use-case for it?

File: app/templates/route-components/avatars.hbs [hunk 7]
```diff
@@ -2,10 +2,6 @@
 <p>
 	Avatars across Fandom should be circular with a 2px internal border.
 </p>
-<p>
```
Comment (tomasznapieralski) [inline app/templates/route-components/avatars.hbs]: It would be added in https://wikia-inc.atlassian.net/browse/XW-2213

**Pull request merged**